ament_index_cpp/CMakeLists.txt

```diff
@@ -38,6 +41,7 @@ install(DIRECTORY include/
   DESTINATION include)
 
 install(TARGETS ${PROJECT_NAME}
+  EXPORT export_${PROJECT_NAME}
```

Comment:
Is this necessary, given the code in the `ament_export_interfaces` code?

https://github.com/ament/ament_cmake/blob/master/ament_cmake_export_interfaces/cmake/ament_cmake_export_interfaces_package_hook.cmake#L28-L33